variables.tf

```diff
@@ -19,6 +19,11 @@ variable "validation_record_ttl" {
   description = "Route 53 time-to-live for validation records"
 }
 
+variable "allow_validation_record_overwrite" {
+  default     = "true"
```

Comment:
I can see an argument for both sides of this default. I chose `true` because setting it to `false` would lead to a major version change (backwards incompatibility) for this module.